pallets/nfts/src/tests.rs

```diff
@@ -4112,168 +4164,212 @@ fn clear_collection_metadata_works() {
 }
 
 #[test]
-fn collection_item_works() {
+fn mint_requires_deposit_works() {
```

Comment:
Yes!! This test makes reviewing the code so much better!! Because we know exactly what is tested and why (by looking at the changes in the main PR), it is much clearer and improves the quality of the review tremendously. 

Like we discussed in the call two weeks ago, we would like to see the tests that already existed with the least amount of changes possible. Then, all the added logic to dispatchables is tested in a separate test function.